src/index.html

```diff
@@ -3,12 +3,41 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        content="
+          width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0
+        "
+  >
```

Comment:
we talked sooo many times about indentations, what's wrong this time?

Reply:
sorry)